lib/talk_helpers.rb

```diff
@@ -6,3 +6,11 @@ def past_and_next_talks items
     i.attributes[:kind] == :talk and i.attributes[:date] <= limit
   end.sort_by { |i| i.attributes[:date] }
 end
+
+def upcoming_talks items
+  limit = Date.today
```

Comment:
Have a closer look at the diff. This is a filter for getting the _upcoming_ talks. So all talks which are in the future. The method does exactly this!